form.html

```diff
@@ -10,7 +10,6 @@
 				<h1>Form</h1>
         <label for="patty">What type of protein would you like?</label>
         <input type="text" name="patty" id="patty">
-        </section>
         <br>
         <span>What toppings would you like?</span>
         <br>
```

Comment:
I have mentioned to use input password type and check box
FYI : i have also informed to add SS in description